### PR TITLE
info for redirect login

### PR DESCRIPTION
**Pull request by mikrdgz**
# [DEVDOCS-1100](https://jira.bigcommerce.com/browse/DEVDOCS-1100)

## What changed?
* Added a new section to Developing Further for modifying forms and included information on redirecting customer upon login

### CODE REVIEW

File: docs/stencil-docs/developing-further/modifying-forms.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+"new_customer": {
+            "heading": "New Customer?",
+            "intro": "Create an account with us and you'll be able to:",
+            "fact1": "Get customer perks",
+            "fact2": "Place a recurring order"
+        }
+```
+
+### Login Form Redirect
+By default, customers are redirected to `/account.php` upon login, but you have the option to redirect customers to another relative URL. You can do this by passing the parameter `id="redirect_to"` with a `value={path}` on a hidden form field within the `<form>` element in the template. For example
```
Comment (uwikaiddi) [inline docs/stencil-docs/developing-further/modifying-forms.md]: Suggestion:

For example **:**

File: docs/stencil-docs/developing-further/modifying-forms.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Modifying Login Form
+<div class="otp" id="no-index">
+
+### On This Page
+- [Modifying The Template](#modifying-the-template)
+</div>
+
+## Modifying The Template
+You can customize the login experience by modifying your theme's `templates/pages/auth/login.html` file.
```
Comment (aglensmith) [inline docs/stencil-docs/developing-further/modifying-forms.md]: > by modifying your theme's

Change `your` to `the` -- We shouldn't assume who owns the theme, in this case.

File: docs/stencil-docs/developing-further/modifying-forms.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+```
+You will find key value pairs for default form text in `/assets/lang/en.json`, and can modify them with your own custom values. For example:
+
+```json
+"new_customer": {
+            "heading": "New Customer?",
+            "intro": "Create an account with us and you'll be able to:",
+            "fact1": "Get customer perks",
+            "fact2": "Place a recurring order"
+        }
```
Comment (aglensmith) [inline docs/stencil-docs/developing-further/modifying-forms.md]: This json hurts on my eyes. Should be formatted to look like this: 

```json
"new_customer": {
       "heading": "New Customer?",
       "intro": "Create an account with us and you'll be able to:",
       "fact1": "Get customer perks",
       "fact2": "Place a recurring order"
 }
```

File: docs/stencil-docs/developing-further/modifying-forms.md (hunk 1 above)
Comment (aglensmith) [inline docs/stencil-docs/developing-further/modifying-forms.md]: Are there any other parameters that can be used on the login form we might want to document here? If there are, we should consider creating a table of the parameters and their descriptions as a comprehensive reference.

**Pull request merged**